library/src/main/java/hotchemi/android/rate/DialogManager.java

```diff
@@ -18,9 +18,20 @@ private DialogManager() {
 
     static Dialog create(final Context context, DialogOptions options) {
         AlertDialog.Builder builder = getDialogBuilder(context);
-        builder.setMessage(options.getMessageResId());
+        if(options.getMessageText() == null) {
```

Comment:
Yup exactly and your change in this class is not formatted(include other place). Please format again with android studio default setting.